userspace/libscap/scap_suppress.c

```diff
@@ -150,9 +150,16 @@ int32_t scap_update_suppressed(struct scap_suppress *suppress,
 
 	HASH_FIND_INT64(suppress->m_suppressed_tids, &ptid, stid);
 
+	/* We suppress the acutal tid in 2 cases:
+	 * 1. Its parent is suppressed.
+	 * 2. Its comm is in the suppressed list
+	 */
 	if(stid != NULL)
 	{
 		*suppressed = true;
+		/* Please note that in this case we don't update the suppressed_comms 
+		 * even if the comm of the child could be different from the parent one.
+		 */
```

Comment:
We never really update suppressed_comms (except for an explicit consumer API call)
```suggestion
		/* Please note that we don't update the suppressed_comms 
		 * even if the comm of the child could be different from the parent one.
		 */
```

Reply:
ty!